src/libraries/System.Speech/src/System.Speech.csproj

```diff
@@ -0,0 +1,256 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;netcoreapp2.0-windows;netstandard2.0</TargetFrameworks>
+    <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
+    <!-- CS0649: uninitialized interop type fields -->
+    <!-- CA2249: Consider using String.Contains instead of String.IndexOf.  We compile for netcoreapp2.0 where this doesn't exist.-->
+    <!-- SA1129: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3277 -->
+    <NoWarn>$(NoWarn);CS0649;CA2249;SA1129</NoWarn>
+    <!-- On older TFM's we needed to disable obsoletion warnings about Marshalling VARIANTs -->
+    <NoWarn Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)-windows'">$(NoWarn);IDE0079</NoWarn>
```

Comment:
properties that depend on `TargetFramework` need to be on a separate `PropertyGroup` after this `PropertyGroup` to work properly on DesignTime builds.

Reply:
Is this still the case? @Anipik @ViktorHofer ?

Reply:
yes they need to be in second or later property group